python/fusion_engine_client/messages/device.py

```diff
@@ -0,0 +1,47 @@
+import struct
+
+from construct import (Struct, Int64ul, Int16ul, Int8ul, Int16sl, Padding, this, Bytes, PaddedString)
+
+from ..utils.construct_utils import AutoEnum, construct_message_to_string
+from ..utils.enum_utils import IntEnum
```

Comment:
Same here - you're not using enums so don't need `IntEnum` or `AutoEnum`

Reply:
Just as a general note, PyLint can show you which imports are being used. In VSCode it will grey them out to make it easy to prune https://stackoverflow.com/questions/53357649/detect-unused-imports-in-visual-studio-code-for-python-3 .